### PR TITLE
Fix create_item reporter

### PR DESCRIPTION
**Pull request by gakada**
Related: https://github.com/poooi/poi/pull/2209.

This can be implemented a bit differently, to differentiate between regular and bulk development (say, if they can differ in results or rates), but that would require updating the server schema as well.

### CODE REVIEW

**Comment by KagamiChan** [comment]
It seems no longer necessary to report failures since the item could not be predicted

**Comment by KagamiChan** [comment]
OK, I think failed reports still make sense

File: reporters/create-item.es [hunk 1]
```diff
@@ -5,20 +5,19 @@ export default class CreateItemReporter extends BaseReporter {
...
-          parseInt(postBody.api_item4),
-        ],
-        itemId:
-          body.api_create_flag == 1
-            ? body.api_slot_item.api_slotitem_id
-            : parseInt(body.api_fdata.split(',')[1]),
-        teitokuLv: _teitokuLv,
-        secretary: _ships[secretaryIdx].api_ship_id,
-        successful: body.api_create_flag == 1,
+      body.api_get_items.forEach(e => {
```
Comment (KagamiChan) [inline reporters/create-item.es]: prefer a meaningful name, like `item`

Comment (gakada) [reply reporters/create-item.es]: changed to `item`

**Comment by KagamiChan** [comment]
Thanks

**Pull request merged**